extensions/http-curl/processors/InvokeHTTP.h

```diff
@@ -150,7 +150,7 @@ class InvokeHTTP : public core::Processor {
       .withDefaultValue("false")
       .build();
   EXTENSIONAPI static constexpr auto DisablePeerVerification = core::PropertyDefinitionBuilder<>::createProperty("Disable Peer Verification")
-      .withDescription("Disables peer verification for the SSL session")
+      .withDescription("DEPRECATED. Only kept for backwards compatibility, no functionality is included.")
```

Comment:
```suggestion
      .withDescription("DEPRECATED. The value is ignored, peer and host verification are always performed when using SSL/TLS.")
```

Reply:
Updated in 55a700da1079435db2a352da9e672c6fa5ce2501